testplans/docker-images/Dockerfile.oni-buildbase

```diff
@@ -1,10 +1,10 @@
-ARG GO_VERSION=1.14.2
+ARG GO_VERSION=1.15.5
 
 FROM golang:${GO_VERSION}-buster
 
 RUN apt-get update && apt-get install -y ca-certificates llvm clang mesa-opencl-icd ocl-icd-opencl-dev jq gcc git pkg-config bzr libhwloc-dev
 
-ARG FILECOIN_FFI_COMMIT=1985275547f222e8c97a8ab70b5cc26bc1fa50b1
+ARG FILECOIN_FFI_COMMIT=1d9cb3e8ff53f51f9318fc57e5d00bc79bdc0128
```

Comment:
We might want to extract this from the Docker image and compile on every test run. `filecoin-ffi` library is changing much more often than originally anticipated, so the fact that we have to maintain the `correct` version in the `base` image we use for the testplan is not ideal.

Reply:
> We can rely on docker caching.

Not sure what you mean with that. Could you elaborate?

---

I think we should just rely on Lotus and clone Lotus for a given commit. We discarded that option early on, but time and time again Lotus is changing underneath these tests and we have to replicate the Lotus setup - for example we no longer need `blst` to be included as an `extern`.